front/app/containers/Admin/tools/Esri/index.tsx

```diff
@@ -31,16 +31,16 @@ const Esri = () => {
         h="240px"
         background={colors.teal200}
         display="flex"
-        justifyContent="center"
+        // justifyContent="center"
       >
         <Box
           background={colors.white}
           display="flex"
           p="10px"
-          width="100%"
-          borderRadius="5px"
-          justifyContent="center"
-          alignContent="center"
+          // width="100%"
+          // borderRadius="5px"
```

Comment:
Not sure about this line actually. To be checked and compared with other items in the tools list.